README.adoc

```diff
@@ -285,6 +285,104 @@ spec:
 ...
 ----
 
+== Configure Health
+
+If you want customize the health alerts in Kiali you can do it in the configuration yaml.
```

Comment:
I would remove this sentence as the section title `Configure Health` is self-explanatory.